lib/thread.c

```diff
@@ -1240,6 +1234,13 @@ thread_fetch (struct thread_master *m, struct thread *fetch)
   return NULL;
 }
 
+static unsigned long
+timeval_elapsed (struct timeval a, struct timeval b)
```

Comment:
Isn't this monotime_since?  If not shouldn't we use an appropriate function in monotime.h?